src/software/ai/motion_constraint/motion_constraint_visitor.cpp

```diff
@@ -85,6 +85,15 @@ void MotionConstraintVisitor::visit(const MoveGoalieToGoalLineTactic &tactic)
     });
 }
 
+void MotionConstraintVisitor::visit(const BallPlacementTactic &tactic)
+{
+    current_allowed_constraints = std::set<TbotsProto::MotionConstraint>();   
+    for (int i = TbotsProto::MotionConstraint_MIN; i < TbotsProto::MotionConstraint_MAX; ++i)
+    {
+        current_allowed_constraints.emplace(static_cast<TbotsProto::MotionConstraint>(i));
+    }
```

Comment:
The only motion constraint that we probably want to _not_ be allowed is the one that includes the goals (don't want to try driving through the goal post). So we should either add one of the motion constraints listed in #2700 (e.g. `AVOID_FIELD_BOUNDARY_ZONE`), or add a TODO here for it to be added as a part of that ticket.

Reply:
done